ui-tests/test/kernels.spec.ts

```diff
@@ -142,4 +142,60 @@ test.describe('Kernels', () => {
     expect(output).toBeTruthy();
     expect(output![0].trim()).toBe('Final check');
   });
+
+  test('Stdin using pyodide kernel', async ({ page }) => {
```

Comment:
Looks like the Firefox failure may be because the Pyodide kernel takes some time to start:

![image](https://github.com/user-attachments/assets/4460d3f0-78a4-44d8-bedd-f414655cd772)

[jupyterlite-firefox-ui-tests.webm](https://github.com/user-attachments/assets/5f620ed3-2674-4028-81c0-216b639005f9)

So maybe it could be worth trying with a higher timeout like above?

```ts
test.setTimeout(120000);
```

Reply:
I'll try it.

Reply:
That seems to have worked.